third_party/gimme/gimme

```diff
@@ -314,8 +314,10 @@ _setup_bootstrap() {
 # _compile "dir"
 _compile() {
 	(
-		if grep -q GOROOT_BOOTSTRAP "${1}/src/make.bash" &>/dev/null; then
-			_setup_bootstrap || return 1
+	  if [[ -z "${GOROOT_BOOTSTRAP}" ]]; then
```

Comment:
is there an upstream commit we should sync to instead?

Reply:
I did not see any changes in this area in upstream @BenTheElder. They had only updated the list of versions above.

Reply:
previously this code wasn't actually modified IIRC, only copied locally so we don't have to curl | bash

Reply:
we should probably document any changes we make and consider upstreaming them